packages/react-vapor/src/components/select/SelectConnected.tsx

```diff
@@ -119,28 +101,14 @@ export class SelectConnected extends React.PureComponent<ISelectProps & ISelectS
             open: this.props.isOpened,
             'mod-multi': this.props.multi,
         });
-        const minWidth = (this.dropdown && this.dropdown.clientWidth) || undefined;
+        const minWidth = this.dropdown.current?.clientWidth || undefined;

```

Comment:
you dont use the same object than before. Are you sure. current is available ?

Reply:
https://reactjs.org/docs/refs-and-the-dom.html#creating-refs